src/core/index.ts

```diff
@@ -63,10 +63,31 @@ interface ChannelSubAction {
 
 type PromptAction = ChannelSubAction;
 
-interface BellPromptConfig {
+export enum ReminderTimeUnit {
+    MINUTES = 'mins',
```

Comment:
Think we can _probably_ drop minutes for reminders?

Guess not doing harm having it, just don't see a use case where 1 hour might not suit instead. (Could be wrong)

Reply:
Yeah, did mull over adding it (was originally just days without even the hours) - had gone with the 'cover unknown cases' approach, but can certainly remove - will leave any changes to last for further thoughts on whether to chop or not 👍

Reply:
Yep, cool. Can see a case for hours, like reasonably a few hours => next 'session'.